src/graphics/shaderloader.cpp

```diff
@@ -0,0 +1,79 @@
+#include "graphics/shaderloader.h"
+
+#include <stdio.h>
+#include <iostream>
+#include <fstream>
+#include <string>
+
+using namespace nautical;
+using namespace nautical::graphics;
+
+std::string* loadFile(const char* fileLoc)
+{
+    std::string* file = new std::string();
```

Comment:
<_<

RAII would be preferable in this instance, and make the code a bit cleaner.